testing/run

```diff
@@ -25,25 +25,32 @@ prechecks() {
             false
         fi
 
-        export ARTIFACT_DIR="/tmp/ci-artifacts_$(date +%Y%m%d)"
+        if [[ -z ${1:-} ]]; then
+            # Default osde2e artifacts dir
```

Comment:
Would be nice to document why it's the default:
`# Even though this repo is not mainly used for osde2e, it was decided to have artifacts dir used by osde2e as the default, because passing arguments is limited in osde2e`

Reply:
Added